app/sidekiq/form526_status_polling_job.rb

```diff
@@ -20,7 +20,7 @@ def perform
     submissions.each_slice(max_batch_size) do |batch|
       batch_ids = batch.pluck(:backup_submitted_claim_id).flatten
       response = api_to_poll.get_bulk_status_of_uploads(batch_ids)
-      handle_response(response)
+      handle_response(response, batch)
```

Comment:
we are creating a potential race condition here where the submission object is held in memory, then we make the API call, then update the existing object. I think it would be safer to itterate over `response.body['data']` and look use activerecord to look up each submission by it's `backup_submitted_claim_id`  inside the loop. 🤔 that's potentially way more DB calls, but it feels like best practice.  idk what do you think?

Reply:
I was attempting to solve for the hundreds of submissions that never seem to be updated by the polling job, extending back to 5/18, by emulating the approach taken in [BenefitsIntakeStatusJob](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/sidekiq/benefits_intake_status_job.rb). That method may not be the best either, but I was hoping improve upon the current situation because, it seems, the states of submissions are not being updated, even though the status responses being returned by the service are either 'success' or 'vbms'. 

Happy to consider a different approach, however.